ports/minizip-ng/portfile.cmake

```
@@ -8,11 +8,25 @@ vcpkg_from_github(
         Modify-header-file-path.patch
 )
 
+vcpkg_check_features(
+    OUT_FEATURE_OPTIONS FEATURE_OPTIONS
+    FEATURES
+        crypto MZ_PKCRYPT
+        crypto MZ_SIGNING
+        crypto MZ_WZAES

```

```suggestion
        pkcrypt MZ_PKCRYPT
        signing MZ_SIGNING
        wzaes MZ_WZAES
```
This might also be updated.

---

😲 sorry, that was sloppy.